i18n/de/xdg_desktop_portal_cosmic.ftl

```diff
@@ -0,0 +1,15 @@
+allow = Erlauben
+cancel = Abbrechen
+capture = Aufnehmen
+share = Teilen
+save-to = Speichern in
+    .clipboard = { save-to } Zwischenablage
+    .pictures = { save-to } Bilder
+    .documents = { save-to } Dokumente
```

Comment:
This is kinda clunky in German. This works better, even though it can't easily be combined with `save-to`.

```suggestion
    .clipboard = In Zwischenablage speichern
    .pictures = In Bilder speichern
    .documents = In Dokumente speichern
```